crates/swc_css_lints/src/rules/mod.rs

```diff
@@ -53,6 +55,9 @@ pub fn get_rules(LintParams { lint_config }: &LintParams) -> Vec<Box<dyn LintRul
         (&rules_config.color_no_invalid_hex).into(),
     ));
     rules.push(unit_no_unknown((&rules_config.unit_no_unknown).into()));
+    rules.push(selector_max_combinators(
```

Comment:
Oh wait, I think this will make linter always run regardless of config.
I missed it in previous review.

Reply:
You're right, but I think we should fix it in another PR?

Reply:
Yeah, you are right, thanks!